liota/device_comms/cip_ethernet_ip_device_comms.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
```

Comment:
rename cip_device_comms.py

Reply:
CIP could be implemented over four different networks.
I have done it over ethernet, so It is known as EtherNet/IP, and thus me and Piyush named it as cip_ethernet_ip.
Is it necessary to rename?